v2.6/getting-started/kubernetes/installation/integration.md

```diff
@@ -216,37 +216,3 @@ kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/instal
 ```
 
 [Click here to view the above yaml directly.](rbac.yaml)
-
-## Configuring Kubernetes
-
-### Configuring the Kubelet
-
-The Kubelet needs to be configured to use the Calico network plugin when starting pods.
-
-The `kubelet` can be configured to use Calico by starting it with the following options
-
-- `--network-plugin=cni`
-- `--cni-conf-dir=/etc/cni/net.d`
-- `--cni-bin-dir=/opt/cni/bin`
-
-For Kubernetes versions prior to v1.4.0, the `cni-conf-dir` and `cni-bin-dir` options are
-not supported.  Use `--network-plugin-dir=/etc/cni/net.d` instead.
-
-See the [`kubelet` documentation](https://kubernetes.io/docs/reference/generated/kubelet/)
```

Comment:
Personally I'd vote for just removing this line - I think the other bits of information are useful and not represented elsewhere in our docs for these older releases.